tests/test_basic.py

```diff
@@ -297,3 +297,71 @@ def test_mons_up_down(rook_cluster):
         check += 1
 
     assert mon_pods == mons
+
+
+def test_rbd_pvc(rook_cluster):
+    # create a CephBlockPool
+    output = rook_cluster.kubernetes.kubectl_apply(
+                os.path.join(
+                    rook_cluster.ceph_dir, 'csi/rbd/storageclass.yaml'))
+
+    if output[0] != 0:
+        pytest.fail("Could not create a CephBlockPool StorageClass")
+
+    # check if StorageClass is up and available
+    output = rook_cluster.kubernetes.kubectl('get sc rook-ceph-block')
```

Comment:
I wonder if there is a chance of a race here where the `kubectl apply` is still working?

Reply:
You are correct.
I am using `wait_for_result instead` now.